pom.xml

```diff
@@ -165,6 +165,20 @@
                 <artifactId>cdm-core</artifactId>
                 <version>${cdm.version}</version>
                 <scope>${cdm.scope}</scope>
+                <exclusions>
+                    <exclusion>
+                        <groupId>com.google.guava</groupId>
+                        <artifactId>guava</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.httpcomponents</groupId>
+                        <artifactId>httpclient</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>com.google.protobuf</groupId>
+                        <artifactId>protobuf-java</artifactId>
+                    </exclusion>
```

Comment:
Note: `protobuf-java` became shaded since Spark 3.5 ([SPARK-41244](https://issues.apache.org/jira/browse/SPARK-41244)). However, protobuf is only used for the experimental [NetCDF streaming format](https://docs.unidata.ucar.edu/netcdf-java/5.6/userguide/ncstream.html) so excluding it won't affect our NetCDF support.